sqlmesh/dbt/common.py

```diff
@@ -176,21 +176,18 @@ class Dependencies(PydanticModel):
     """
 
     macros: t.List[MacroReference] = []
```

Comment:
What's the purpose of macros remaining as a list, while the others are now sets?

Reply:
you can use a frozenset if you want hashability

Reply:
How `frozenset` helps with hashability? The problem is that `MacroReference` is serialized into dict which is not hasbable. Set is not a problem